services/movie_session.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime, movie_id: int, cinema_hall_id: int):
```

Comment:
Where are type annotations for returned value?

Reply:
Check lattest commit, please.